tests/hubspot/test_hubspot.py

```diff
@@ -79,17 +100,77 @@ def test_hubspot_wrong_data_return(connector, datasource, mocker):
 
 def test_hubspot_get_webanalytics(connector, datasource, mocker):
     req_mock = mocker.patch('requests.get')
-    req_mock.return_value.json = lambda: {'results': [{'properties': [{'foo': 42}]}]}
+    req_mock.return_value.json = lambda: {
+        'results': [
+            {
+                'properties': {'foo': 42},
+                'eventType': 'party',
+                'occuredAt': '2021-03-08T12:28:40.305Z',
+            }
+        ]
+    }
 
     datasource.object_type = HubspotObjectType.contact
     datasource.dataset = HubspotDataset.webanalytics
 
-    connector.get_df(datasource)
+    df = connector.get_df(datasource)
 
     assert req_mock.called_once()
     called_endpoint, kwargs = req_mock.call_args
-    assert called_endpoint[0] == HUBSPOT_ENDPOINTS['web-analytics']
+    assert called_endpoint[0] == HUBSPOT_ENDPOINTS['web-analytics']['url']
     assert kwargs['params'] == {'objectType': HubspotObjectType.contact}
+    assert not df.empty
+    assert df[0]['eventType'] == 'party'
+    assert df[0]['occuredAt'] == '2021-03-08T12:28:40.305Z'
+
```

Comment:
Pour tester que tu as bien une dataframe comme dans les email events, tu peux echanger : 

```suggestion
    assert df['eventType'][0] == 'party'
    assert df['occuredAt'][0] == '2021-03-08T12:28:40.305Z'

```

Reply:
Fixé ici : https://github.com/ToucanToco/toucan-connectors/commit/f75b0319ab910501e34378b4a23036997c6a3853#diff-890de93d94e4a0fa6918e0a3ff652f2c84a9b6ebe90eae80d354665a28c5ff3aL123-L124